### PR TITLE
Add LICENSE, NOTICE, README and such

### PR DESCRIPTION
**Pull request by BigBlueHat**
Stuff and things.
🎩

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "{}"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright {yyyy} {name of copyright owner}
```
Comment (Treora) [inline LICENSE:189]: forgotten?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Selector State Fragments
+
+Converts Web Annotation
+[Selectors and States as Fragment Identifiers](https://www.w3.org/TR/selectors-states/#frags)
+into a [Web Annotation Data Model Selector](https://www.w3.org/TR/selectors-states/#selectors).
```
Comment (Treora) [inline README.md]: More accurately, we create a Specific Resource, right? Which can *contain* a selector, or a state.

Comment (Treora) [reply README.md]: Also, it can also transform it vice versa.

File: package.json [hunk 3]
```diff
@@ -3,6 +3,15 @@
   "version": "0.1.0",
   "description": "Converter between the JSON and the URI+Fragment representation of Selectors and States",
   "main": "lib/index.js",
+  "contributors": [
+    {
+      "name": "Ivan Herman",
+      "email": "ivan@w3.org"
+    },
+    {
+      "name": "Gerben"
```
Comment (Treora) [inline package.json]: gerben@treora.com is fine.

**Review by Treora — COMMENTED** [review]
Thanks! Feel free to also suggest a better module name if you have an idea.

File: LICENSE (hunk 1 above)
Comment (BigBlueHat) [reply LICENSE:189]: Yep. Will fix.

File: README.md (hunk 2 above)
Comment (BigBlueHat) [reply README.md]: True.

File: LICENSE (hunk 1 above)
Comment (BigBlueHat) [reply LICENSE:189]: Actually, no. That's in the appendix. AFAIK having the license in the LICENSE file is sufficient, but the appendix talks about adding that header to each file...I know the ASF is picky about that, but not sure how mandatory it is. Regardless, the appendix doesn't need changing. 😄

**Comment by BigBlueHat** [comment]
@Treora updates made.

**Comment by BigBlueHat** [comment]
@Treora looks like the header text is a SHOULD--so we'll skip that one for now:
http://www.apache.org/dev/apply-license.html#new

**Pull request merged**